server/src/infra/repositories/person.repository.ts

```diff
@@ -10,6 +11,53 @@ export class PersonRepository implements IPersonRepository {
     @InjectRepository(AssetFaceEntity) private assetFaceRepository: Repository<AssetFaceEntity>,
   ) {}
 
+  async updateAssetsId(primaryPersonId: string, mergePersonId: string, assetIds: string[]): Promise<number> {
+    const result = await this.assetFaceRepository
+      .createQueryBuilder()
+      .update()
+      .set({ personId: primaryPersonId })
+      .where({ assetId: In(assetIds), personId: mergePersonId })
```

Comment:
Why do you need a list of assets IDs? It should be _every_ record with merge person id.